readme.rst

```diff
@@ -22,3 +25,6 @@ Todo
 - improve documentation
 
   - structure
+
+  .. |GitHub Actions| image:: https://github.com/EVerest/libevse-security/actions/workflows/build_and_test.yml/badge.svg
+      :target:
```

Comment:
What is the expected result here? Seems to not work, for me at least. For me, it would be okay to omit it.